controllers/main_controller.rb

```diff
@@ -9,3 +9,19 @@
   # combined document back to the client.
   erb :"main/welcome"
 end
+
+MyApp.get "/about" do
+  erb :"main/about"
+end
+
+MyApp.get "/favorites" do
+  erb :"main/favorites"
+end
+
+MyApp.get "/greet/Sumeet" do
```

Comment:
Yes, just for the record I didn't actually delete them, but I commented them out. I'll re-push right now.